arcon_allocator/src/lib.rs

```diff
@@ -56,6 +59,13 @@ pub struct Allocator {
 impl Allocator {
     /// Creates a new Allocator with the given memory limit size
     pub fn new(limit: usize) -> Allocator {
+        #[cfg(feature = "allocator_metrics")]
+        {
+            register_gauge!("total_bytes", "arcon_allocator" => "arcon_allocator");
```

Comment:
Do we need the "arcon_allocator" => "arcon_allocator" part?

```rust
register_gauge!("bytes_remaining",  "allocator");
// or 
register_gauge!("allocator_bytes_remaining");


```

Reply:
I added that to maintain consistency with counters registration as per the node metrics